model/database/metrics.yaml

```diff
@@ -133,3 +134,39 @@ groups:
     attributes:
       - ref: db.client.connection.pool.name
         requirement_level: required
+
+  - id: metric.db.cosmosdb.operation.request_charge
+    type: metric
+    metric_name: db.cosmosdb.operation.request_charge
+    brief: "Request Charge consumed by the operation"
+    instrument: histogram
+    unit: "{request_units}"
+    stability: experimental
+    extends: attributes.db.cosmosdb.minimal
+
+  - id: metric.db.cosmosdb.operation.max_item_count
+    type: metric
+    metric_name: db.cosmosdb.operation.max_item_count
+    brief: "Max item count requested by the feed operation (ex. Query, ReadAll, ReadMany, Change Feed) "
+    instrument: gauge
+    unit: "{count}"
+    stability: experimental
+    extends: attributes.db.cosmosdb.minimal
+
+  - id: metric.db.cosmosdb.operation.actual_item_count
+    type: metric
+    metric_name: db.cosmosdb.operation.actual_item_count
+    brief: "Actual item count returned by the feed operation (ex. Query, ReadAll, ReadMany, Change Feed)"
+    instrument: gauge
+    unit: "{count}"
+    stability: experimental
+    extends: attributes.db.cosmosdb.minimal
+
+  - id: metric.db.cosmosdb.operation.regions_contacted
+    type: metric
+    metric_name: db.cosmosdb.operation.regions_contacted
+    brief: "Number of regions contacted for the operation"
+    instrument: gauge
```

Comment:
you probably want it as a histogram. Gauge is designed to report something like temperature that does not change much - it reports the last value returned during the interval. 

If histogram is too expensive/complicated, simple counter can give users an average number of regions per operation

Reply:
Changed all the metrics to histogram, it gives us more flexibility to do diff calculations.